pom.xml

```diff
@@ -106,6 +106,11 @@
         <artifactId>graphhopper-reader-osm</artifactId>
         <version>0.13.0-pre13</version>
       </dependency>
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-api</artifactId>
+        <version>5.6.0</version>
```

Comment:
this seems like fixing a previous unnoticed problem with using junit from transitive dependency resolution?

Reply:
Transitive dependencies are intentional when using Spring Boot [Starters](https://docs.spring.io/spring-boot/docs/2.2.4.RELEASE/reference/html/using-spring-boot.html#using-boot-starter).

This dependency has to be added because since [2.2.0.RELEASE](https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-test/2.2.0.RELEASE), `junit-jupiter-api` is part of `spring-boot-starter-test`. Downgrading Spring Boot to 2.1.x made the dependency disappear because `spring-boot-starter-test` [2.1.12.RELEASE](https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-test/2.1.12.RELEASE) doesn't have it.

Reply:
Correction: the comment above is not relevant. `optaweb-vehicle-routing-backend` declares `junit-jupiter-api` dependency so it doesn't use it transitively.

The dependency this PR introduces is a *dependency management* that overrides the version to the latest. This is necessary because one of the tests uses JUnit's `@TempDir` annotation that was introduced in [5.4.0](https://junit.org/junit5/docs/5.4.0/release-notes/).

Reply:
just in case you are aware we use 5.5.2 version in other related kiegroup depMgmt, isn't this confusing to mix more minor versions?

Reply:
@mareknovotny In that case I can change this dependency management version to 5.5.2. Would that address this concern?

Reply:
Yes, please `${version.org.junit}` from `kie-parent` instead of hard-coding 5.6.0. In this case I think it makes sense because 5.5.2 is the version used by [spring-boot-starter-test:2.1.12.RELEASE](https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-test/2.1.12.RELEASE).

Reply:
Version overrides are tricky. The tests probably don't run because of version mismatch between Jupiter API and the engine. Not sure, but I only managed to run tests with JUnit 5.3.2, which is the version that you get with no overrides.

Please remove this once #273 is merged (it makes the downgrade to 5.3.2 possible by avoiding `@TempDir`, which is 5.4.0+).